react/__tests__/components/ProductPrice.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import ProductPrice from './../../ProductPrice'
+import { render } from 'enzyme'
+
+describe("<ProductPrice />", () => {
+  let renderComponent = null
+  const defaultProps = {
+    loaderClass: '',
+    showListPrice: false,
+    sellingPrice: 40,
+    intl: { formatNumber: (number) => number }
+  }
+  
+  const context = { culture: { currency : 'USD' } };
+
+  beforeEach(() => {
+    renderComponent = customProps => {
+      const props = {
+        ...defaultProps,
+        ...customProps,
+      }
+      return render(<ProductPrice {...props} />, { context })
+    }
+  })
+
+  it('should be mount', () => {
+    expect(renderComponent()).toBeDefined()
+  })
+
+  it('should match snapshot', () => {
+    expect(renderComponent()).toMatchSnapshot()
```

Comment:
It's more useful and clear when instead of using snapshot, we expect to find a specific element on the rendered DOM.